front/src/pages/internal/profile/availabilities/AvailabilitiesUpdatePage.tsx

```diff
@@ -0,0 +1,285 @@
+    const category = categories?.find((c) => c.id === value);
+    handleInputChange('categoryId', value);
+    handleInputChange('category', category?.category);
+  };
+
+  return (
+    <ScrollView
+      style={styles.container}
+      contentContainerStyle={styles.contentContainer}
+    >
+      <View style={styles.verticalCenterContainer}>
+        <View style={styles.horizontalContainer}>
+          <View>
+            <Text>{i18n.t('profile.date.dateRange')}</Text>
+            <Text>
+              {`${new Date(formData.startDate).toLocaleDateString(
+                'en-US',
+              )} - ${new Date(formData.endDate).toLocaleDateString('en-US')}`}
+            </Text>
+          </View>
+          <Button
+            onPress={() => setOpen(true)}
+            uppercase={false}
+            mode='outlined'
+          >
+            {i18n.t('profile.date.pickRange')}
+          </Button>
+          <DatePickerModal
+            locale='en'
+            mode='range'
+            visible={open}
+            onDismiss={onDismiss}
+            startDate={dateRange.startDate}
+            endDate={dateRange.endDate}
+            onConfirm={onConfirm}
+          />
+        </View>
+        <View style={{ width: '80%', marginTop: 10, marginBottom: 10 }}>
+          {categories && (
+            <DropDown
+              label={'Category'}
+              visible={showDropDown}
+              showDropDown={() => setShowDropDown(true)}
+              onDismiss={() => setShowDropDown(false)}
+              value={formData.categoryId}
+              setValue={(value) => {
+                dropdownChange(value);
+              }}
+              list={categories?.map((category) => ({
+                label: category.category,
+                value: category.id,
+              }))}
+            />
+          )}
+        </View>
+        <TextInput
+          label={i18n.t('profile.address.firstLine')}
+          value={formData.firstLine || ''}
+          style={styles.textInput}
+          onChangeText={(value) => handleInputChange('firstLine', value)}
+        />
+        <View style={styles.horizontalContainer}>
+          <TextInput
+            label={i18n.t('profile.address.zipCode')}
+            value={formData.zipCode || ''}
+            style={styles.smallInput}
+            onChangeText={(value) => handleInputChange('zipCode', value, true)}
+          />
+          <TextInput
+            label={i18n.t('profile.address.city')}
+            value={formData.city || ''}
+            style={styles.smallInput}
+            onChangeText={(value) => handleInputChange('city', value)}
+          />
+        </View>
+        <View
+          style={[
+            styles.horizontalContainer,
+            { width: '80%', marginTop: 20, marginBottom: 20 },
+          ]}
+        >
+          <Text variant='headlineSmall'>
+            {i18n.t('profile.availabilities.radiusRange')} {formData.range[0]}{' '}
```

Comment:
Instead of doing it manually, use the [interpolation method](https://github.com/fnando/i18n?tab=readme-ov-file#translating-messages) from our i18n library.